app/controllers/passwords_controller.rb

```diff
@@ -8,13 +8,13 @@ def update
     @user = current_user
     @user.update(password_params)
     if password_params[:password] != password_params[:password_confirmation]
-      flash[:failure] = "Password and confirmation must match."
+      flash[:error] = "Password and confirmation must match."
       render :edit
     elsif @user.save
       redirect_to '/profile'
-      flash[:error] = "Your password has been updated"
+      flash[:success] = "Your password has been updated"
     else
-      flash[:failure] = "You are missing required fields."
+      flash[:error] = "You are missing required fields."
       render :edit
```

Comment:
Good catch here - I think there may be other places in the app (like where I've written code) we can tighten up our naming conventions on these.